xml/docu_styleguide.language.xml

```diff
@@ -240,6 +240,11 @@
 
  <sect2 xml:id="sec-heading">
   <title>Headings</title>
+  <para>
+   When writing a descriptive section, use a noun-based heading title.
+   For example, "Concepts of Potatoes". When writing a task-based section,
+   use a gerund. For example, "Installing Potatoes".
```

Comment:
In SUMA, we use this language (https://github.com/uyuni-project/uyuni-docs/wiki/wug#headings):

> Headings should be as simple as possible, and should always be written in simple present tense (the action is happening right now). Use declarative noun phrases as much as possible.
> 
> For sections that contain a procedure, use the bare infinitive (base) form of the verb followed by the subject. For example: Install SUSE Manager, or Configure the Client.

We use the gerund form only in Procedure titles, so we get three levels, something like this:

> H1 Potatoes
> (Concepts)
> 
> H2 Peel Potatoes
> (Description of the procedure, and how it fits into the process)
> 
> H3 Procedure: Peeling Potatoes
> (Steps)

This was designed to help parallelism between chapters, and also to help orient readers within chapters. Some of our sections get quite long.

Reply:
I actually like this suggestion a lot, and it's a slight deviation from what we're already doing that helps further clarify. THoughts @taroth21 and @sknorr ?

Reply:
I am not sure what is gained with the added complication of using two different styles between sections and procedures to be honest.

Reply:
> I am not sure what is gained with the added complication of using two different styles between sections and procedures to be honest.

It differentiates between the learning behaviour and the doing behaviour. To use the example I gave above, in H2 Peel Potatoes, you are learning about the action of peeling potatoes. It would include things like equipment you require, the environment you should be in, exactly what thing you are acting on. In H3 Procedure: Peeling Potatoes, it's just doing the thing, steps only, lots of simple verbs: "open this, do that, click there" etc.

Reply:
Hm, ok. How many readers would you expect to notice this subtle change and how do you think it would influence their behavior? I can't really see how that improves the documentation, it seems like mental overhead for writers and it creates extra work/cost during translation.

Reply:
The whole point of good content design is that readers *don't* notice it. The real trick here is consistency: when you do it the same way in every chapter, across every book, it's grounding for the reader and gives a sense of predictability and familiarity. I would argue that this level of consistency is also good for writers (it takes the effort out of coming with titles) and translators (they know how the words around the noun need to be translated in every chapter).